src/routes/+layout.svelte

```diff
@@ -203,49 +203,69 @@
 	{#if envPublic.PUBLIC_APPLE_APP_ID}
 		<meta name="apple-itunes-app" content={`app-id=${envPublic.PUBLIC_APPLE_APP_ID}`} />
 	{/if}
+	<!-- TODO: remove -->
+	{#if !$page.data.embeddedAssistantId}
+		<script
+			src="http://localhost:5173/chat/api/assistant/{$page.data.assistants.at(-1)
+				._id}/embed-snippet"
+			defer
+		></script>
+	{/if}
```

Comment:
to make testing the embedded assistants easier, I'm embedding hf assistant inside the chat-ui. These lines would be removed before merging to `main`.

you can change `{$page.data.assistants.at(-1)._id}` this part that grabs the last assistant id to any specific local assistant id if you want